README.md

```diff
@@ -31,17 +28,15 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Vet clinic db <a name="about-project"></a>
 
-**[your_project__name]** is a...
+Here you can have a store with the name, date of birth, neutered, escape attemps and weight(kg) of all the animals the veterinary have recieve! 
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+The site was built with PSQL
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack <a name="tech-stack"></a>
```

Comment:
- [ ] Kindly remove  React.js in the Client Tech Stack and Express.js in the Server Tech Stack since we're not using them in the project 👇🏻 

![image](https://github.com/enmanuelbayen/Vet-clinic-DB/assets/79058364/4c79bb25-5a19-43e5-b200-ee48874307c7)